.travis.yml

```diff
@@ -3,15 +3,15 @@ node_js:
   - 8
   - 10
   - 12
-cache:
-  directories:
-    - node_modules
```

Comment:
Can you explain why this was removed?

Reply:
Caching node_modules directory had unexpected side effects on CI.  In particular, I got the following error:
```bash
The command "eval npm ci  " failed. Retrying, 2 of 3.
```

Removed cache as per: https://github.com/travis-ci/travis-ci/issues/10113#issuecomment-421121316

Reply:
Gotcha. That's unfortunate! Thanks for fixing 🙏